.github/ISSUE_TEMPLATE/bug-report-template.md

```diff
@@ -2,7 +2,7 @@
 name: Notify Bug Template
 about: Create a report to help us improve
 title: 'BUG: '
-labels: Notify
+labels: Notify, bug
```

Comment:
I would also prepopulate "patch".

Reply:
patch is for the pr _template_, maybe, this is just for the _ticket_ itself.